rosdep/base.yaml

```diff
@@ -3543,6 +3559,9 @@ lm-sensors:
   gentoo: [sys-apps/lm_sensors]
   ubuntu: [lm-sensors]
 log4cxx:
+  alpine:
+    source:
+      uri: 'https://github.com/at-wat/alpine-ros-sources/raw/master/log4cxx/log4cxx.rdmanifest'
```

Comment:
We've found that source rosdeps generally end up being more work than their worth. I thought we'd officially deprecated them but I can't find a reference. But either way they're strongly discouraged. If you've tested them as working we can merge them for now until a better solution is available.

Reply:
I agree that using source rosdeps lacks maintainability pretty much.
Registering them to the upstream package repository would be necessary. But until completing them, keeping source rosdeps would be helpful for testing.